### ISSUE TITLE
Random crashes on Windows 2008 Server

### ISSUE THREAD

**Issue by annaparker** [hexchat/hexchat#1312]
I have noticed that HexChat 2.10.2 x64 (plain one, without any plugins) crashing on my Windows 2008 Server R2 64-bit. Tried on few servers, the results are the same. It can crash after 1 hour of start or 1 day after start or 3 days after start, nobody knows when and what's more important - why? ;-)

I have installed Visual C++ 2013 x64 Redistributable from HexChat's site. I have few servers and noticed that problem only on Windows 2008 Server, there's no issues for long sessions (even for months) on Linux/Windows 7/Windows 8.

Problem signature:
  Problem Event Name:	BEX64
  Application Name:	hexchat.exe
  Application Version:	2.10.2.0
  Application Timestamp:	5474c5e6
  Fault Module Name:	MSVCR120.dll
  Fault Module Version:	12.0.21005.1
  Fault Module Timestamp:	524f83ff
  Exception Offset:	000000000006dd3d
  Exception Code:	c0000417
  Exception Data:	0000000000000000
  OS Version:	6.1.7601.2.1.0.272.7
  Locale ID:	1033
  Additional Information 1:	83cd
  Additional Information 2:	83cd1a00e1fb4b6d72b9345980227528
  Additional Information 3:	2d44
  Additional Information 4:	2d4458932ccde33b34731d00366d5ad

![clipboard01](https://cloud.githubusercontent.com/assets/6404127/6593716/543a4714-c7db-11e4-9b8a-2b32050424dc.jpg)

I'm not programmer, but I'm opened for cooperation - the debug, logs, etc - just let me know step by step what I should do.

**Comment by TingPing** [comment]
Try 32bit hexchat. Vista support was dropped because of random crashes like this, perhaps the same cause.

**Comment by annaparker** [comment]
It's a thrid day without crash, so it seems 32-bit version solved problem out. Thanks.

**Comment by annaparker** [comment]
It crashed after 4 days, the same crash as before. Issue reopened.

**Comment by Mikaela** [comment]
I am just curious, why are you running it on Windows *2008 Server*?

**Comment by annaparker** [comment]
@Mikaela Because I have server with Windows 2008 Server installed on.

**Comment by Mikaela** [comment]
So no specific reason?

I am just wondering why not normal desktop OS on normal device as server editions of Windows are expensive and most of software isn't probably tested on them so much especially when the software has nothing to do with server. I would understand if HexChat was able to connect to remote HexChat like Smuxi/Quassel/WeeChat or if you were running ZNC, but plain HexChat just sounds strange to me.

It's your choice whatever you run though.

**Comment by annaparker** [comment]
@Mikaela It's a server, not desktop/home computer. Thanks a lot to allow me to run my favourite IRC client on. ;-)

**Comment by jderickson** [comment]
I am having the same issue on Windows 7 x64. HexChat 2.10.2.0 x64

Same exception code and offset. Same everything based on the screenshot above and the error dialog I am looking at right now.

**Comment by annaparker** [comment]
Yes, I have noticed that problem on Windows 7 x64 as well as on Windows 2008 Server.
